src/util/entities/Guild.ts

```diff
@@ -390,4 +390,11 @@ export class Guild extends BaseClass {
 
 		return guild;
 	}
+
+	toJSON() {
+		return {
+			...this,
+			unavailable: this.unavailable == false ? undefined : true,
+		};
+	}
```

Comment:
spacebar's current architecture means you will never get any unavailable guilds
meaning this piece of code is redundant as it would always return `false`